code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -127,7 +127,7 @@
 	. = ..()
 
 /obj/item/gun/magic/staff/chaos/on_intruder_use(mob/living/user)
-	if(user.anti_magic_check(TRUE, FALSE, FALSE)) // Don't let people with antimagic use the staff of chaos.
+	if(user.can_block_magic()) // Don't let people with antimagic use the staff of chaos.
```

Comment:
Should be `can_cast_magic`, no?

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/64124/commits/93328cb3c9a7ffd83b00edeb792a6b724cc56622